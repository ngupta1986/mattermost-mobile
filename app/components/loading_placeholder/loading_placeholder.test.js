// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({adapter: new Adapter()});

import LoadingPlaceholder from './index.js';

describe('LoadingPlaceholder', () => {
    test('should match snapshot', () => {
        const wrapper = shallow(
            <LoadingPlaceholder/>
        );

        expect(wrapper).toMatchSnapshot();
    });
});
