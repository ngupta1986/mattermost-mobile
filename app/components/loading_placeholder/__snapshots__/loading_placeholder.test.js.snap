// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPlaceholder should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LoadingPlaceholder />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "accessible": true,
      "allowFontScaling": true,
      "children": Array [
        <InjectIntl(FormattedText)
          defaultMessage="Loading"
          id="loading_screen.loading"
        />,
        <AnimatedEllipsis
          animationDelay={300}
          minOpacity={0.4}
          numberOfDots={3}
          style={
            Object {
              "fontSize": 28,
              "letterSpacing": -3,
              "lineHeight": 30,
              "marginLeft": 5,
            }
          }
        />,
      ],
      "ellipsizeMode": "tail",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defaultMessage": "Loading",
          "id": "loading_screen.loading",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "animationDelay": 300,
          "minOpacity": 0.4,
          "numberOfDots": 3,
          "style": Object {
            "fontSize": 28,
            "letterSpacing": -3,
            "lineHeight": 30,
            "marginLeft": 5,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "accessible": true,
        "allowFontScaling": true,
        "children": Array [
          <InjectIntl(FormattedText)
            defaultMessage="Loading"
            id="loading_screen.loading"
          />,
          <AnimatedEllipsis
            animationDelay={300}
            minOpacity={0.4}
            numberOfDots={3}
            style={
              Object {
                "fontSize": 28,
                "letterSpacing": -3,
                "lineHeight": 30,
                "marginLeft": 5,
              }
            }
          />,
        ],
        "ellipsizeMode": "tail",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defaultMessage": "Loading",
            "id": "loading_screen.loading",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "animationDelay": 300,
            "minOpacity": 0.4,
            "numberOfDots": 3,
            "style": Object {
              "fontSize": 28,
              "letterSpacing": -3,
              "lineHeight": 30,
              "marginLeft": 5,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
